docs: Refresh docs for SQL input source (#17031)
apache/druid: docs/ingestion/input-sources.md

## SQL input source

:::info Required extension
To use the SQL input source, you must load the appropriate extension in your `common.runtime.properties` file.
* To connect to MySQL, load the extension [`mysql-metadata-storage`](../development/extensions-core/mysql.md).
* To connect to PostgreSQL, load the extension [`postgresql-metadata-storage`](../development/extensions-core/postgresql.md).

The MySQL extension requires a JDBC driver.
For more information, see the [Installing the MySQL connector library](../development/extensions-core/mysql.md).
:::

The SQL input source is used to read data directly from RDBMS.

> The SQL input source is _splittable_ and can be used by the [parallel task](./native-batch.md), where each worker task will read from one SQL query from the list of queries.

Reviewer: ```suggestion
You can _split_ the ingestion tasks for a SQL input source. When you use the [parallel task](./native-batch.md) indexer?, each worker task reads from one SQL query from the list of queries.
```
Consider rewording for clarity.

The Combining input source lets you read data from multiple input sources.
It identifies the splits from delegate input sources and uses a worker task to process each split.

> Use the Combining input source only if all the delegates are splittable and can be used by the [parallel task](./native-batch.md).

Reviewer: this reads awkwardly 😬  I know you were just fixing case.

Author: ```suggestion
Each delegate input source must be splittable and compatible with the [parallel task type](./native-batch.md).
```
